parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -299,7 +303,7 @@ systemctlEnableAndStart() {
 }
 
 systemctlDisableAndStop() {
-    if [ systemctl list-units --full --all | grep -q "$1.service" ]; then
```

Comment:
what is the issue here?

Reply:
shell-check was giving a warning about the []

Reply:
hmm. I see the similar usage in L139

Reply:
TODO: separate PR for cse